spec/factory_bot/attribute_list_spec.rb

```diff
@@ -123,28 +135,51 @@ def build_association(name)
     FactoryBot::Attribute::Association.new(name, :user, {})
   end
 
-  before do
-    subject.define_attribute(build_ignored_attribute(:comments_count))
-    subject.define_attribute(build_ignored_attribute(:posts_count))
-    subject.define_attribute(build_non_ignored_attribute(:email))
-    subject.define_attribute(build_non_ignored_attribute(:first_name))
-    subject.define_attribute(build_non_ignored_attribute(:last_name))
-    subject.define_attribute(build_association(:avatar))
-  end
-
   it "knows all its #names" do
-    expect(subject.names).to eq [:comments_count, :posts_count, :email, :first_name, :last_name, :avatar]
+    list = FactoryBot::AttributeList.new
+    list.define_attribute(build_ignored_attribute(:comments_count))
```

Comment:
Is there a reason to have more than one of each type in these tests?

Reply:
Probably not.  I suppose this was a stylistic choice made by the original test author.  I noticed this kind of duplication in a couple of tests.  I'll remove it, might make the test easier to grok.